.eslintrc.json

```diff
@@ -1,3 +1,29 @@
 {
-  "extends": "next/core-web-vitals"
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "airbnb",
+        "airbnb-typescript",
+        "next",
+        "prettier"
+    ],
+    "overrides": [
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module",
+        "project": "./tsconfig.json"
+    },
+    "plugins": [
+    ],
+    "rules": {
+      "react/react-in-jsx-scope": "off",
```

Comment:
Perfect - this is what you need for React 17 onwards until they update the rule (which they'll do once R17+ becomes more mainstream)